torch/csrc/jit/codegen/cuda/executor.cpp

```diff
@@ -1260,7 +1262,7 @@ std::vector<at::Tensor> FusionExecutor::runFusion(
 
     bytes_processed_ = 0;
     // Figure how many bytes are inputs, outputs, and temporary buffers
-    for (auto i : c10::irange(args.size())) {
+    for (auto i : c10::irange(num_inputs)) {
```

Comment:
cc @naoyam

Reply:
Thanks. I believe some code cleanup would be due here (not about this particular fix)

Reply:
Apology for this half finished work. I get dragged away in the middle of the refactor.

I do plan on coming back here and clean things up after 1.13.

Reply:
whoops 😳